aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -282,93 +304,77 @@ dotnet ef database update
 
 Examine the *Migrations/{timestamp}_InitialCreate.cs* migration file:
 
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Migrations/20190805165915_InitialCreate.cs?name=snippet)]
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie5/Migrations/20201107014924_InitialCreate.cs?name=snippet)]
 
-The `Up` method creates the Movie table and configures `Id` as the primary key. The `Down` method reverts the schema changes made by the `Up` migration.
+In the preceding code:
+
+* `InitialCreate.Up` creates the Movie table and configures `Id` as the primary key.
+* `InitialCreate.Down` reverts the schema changes made by the `Up` migration.
 
 <a name="test"></a>
 
 ## Test the app
 
-* Run the app and click the **Movie App** link.
+Run the app and select the **Movie App** link.
 
-  If you get an exception similar to one of the following:
+If you get an exception similar to the following, you may have missed the [migrations step](#migration):
 
 # [Visual Studio](#tab/visual-studio)
 
-  ```console
-  SqlException: Cannot open database "MvcMovieContext-1" requested by the login. The login failed.
-  ```
+```console
+SqlException: Cannot open database "MvcMovieContext-1" requested by the login. The login failed.
+```
 
 # [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
 
-  ```console
-  SqliteException: SQLite Error 1: 'no such table: Movie'.
-  ```
+```console
+SqliteException: SQLite Error 1: 'no such table: Movie'.
+```
 
 ---
-  You probably missed the [migrations step](#migration).
-
-* Test the **Create** page. Enter and submit data.
-
-  > [!NOTE]
-  > You may not be able to enter decimal commas in the `Price` field. To support [jQuery validation](https://jqueryvalidation.org/) for non-English locales that use a comma (",") for a decimal point and for non US-English date formats, the app must be globalized. For globalization instructions, see [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/4076#issuecomment-326590420).
-
-* Test the **Edit**, **Details**, and **Delete** pages.
-
-## Dependency injection in the controller
-
-# [Visual Studio](#tab/visual-studio)
-
-Open the *Controllers/MoviesController.cs* file and examine the constructor:
-
-<!-- l.. Make copy of Movies controller (or use the old one as I did in the 3.0 upgrade) because we comment out the initial index method and update it later  -->
 
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie22/Controllers/MC1.cs?name=snippet_1)]
-
-The constructor uses [Dependency Injection](xref:fundamentals/dependency-injection) to inject the database context (`MvcMovieContext`) into the controller. The database context is used in each of the [CRUD](https://wikipedia.org/wiki/Create,_read,_update_and_delete) methods in the controller.
```

Comment:
This needs to be restored. Why was it dropped?

Reply:
Thanks for catching this.  This is right where I ran into an odd cannot merge commit issue.  Just touching particular lines, no matter what I did would cause the error.  I ended up having to remove a section and build and test adding back to determine where it was hanging up.  I ended up accidently missing this section.

I should not have set this PR back from draft.  I thought it would only notify you for another review if I explicitly re-requested a review, sorry.  I was still testing through this set of instructions for all three IDE's/OS to make sure I didn't miss anything.